torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -241,6 +241,9 @@ class TORCH_API TensorExprKernel {
     return custom_lowerings_;
   }
 
+  // Allocate memory for intermediate buffers at compile time.
```

Comment:
Could you please add an explanation how it works? It might be non-obvious that preallocated bufs end up among constants (which is a totally right thing to do here, just not obvious).